repose-aggregator/src/docs/asciidoc/recipes/quick-start.adoc

```diff
@@ -0,0 +1,166 @@
+= Quick Start
+
+With the advent of container technology, *Repose* can be fully encapsulated and run as a service with minimal setup.
+
+By bundling the environment with the software itself, deploying *Repose* has become a much quicker and simpler process.
+Docker boasts security through isolation, and when run on platforms with native support, it requires very little overhead.
+To make everything as easy and versatile as possible, the *Repose* team now publishes Docker images on Docker Hub as part of every release.
+
+== Prerequisites
+Before you can use a *Repose Docker* image, you will need to install and have running a Docker Container Engine.
+The Docker Container Engine is available for almost all operating systems.
+The installation instructions for the Docker Container Engine are outside the scope of this Quick Start guide, but they can be found https://docs.docker.com/engine/installation/[here].
+Since this is a Quick Start guide, only the minimal features required to get up and running are presented here.
+For a complete exploration of what the Docker Container Engine can provide, take a look at the https://docs.docker.com/[official Docker documentation site].
+
+At this point you may be able to execute the `docker run hello-world` test.
+If you can, then you are almost ready to run a basic *Repose Docker* image.
+If you can't, then the steps below may get you up and going.
+
+== Minor updates
+These are a few minor configuration items that may prevent some platforms from even executing the `docker run hello-world` test.
+They are not always well documented in the installation instructions, but are easily remedied.
+
+You will need to get a https://hub.docker.com[Docker Hub] account.
+Once you have that account, you can verify your credentials by logging in using:
+
+- `docker login`
+
+Another issue that may arise is that communication with the Docker Container Engine is denied do to a lack of permissions.
+This can be fixed by adding the user to the `docker` group.
+On most *NIX systems this can be accomplished by issuing the following command as `root` or another elevated user (e.g. `sudo`):
+
+- `usermod -aG docker $USER`
+
+== Confirming you are ready
+Make sure you can successfully run the Docker test:
+
+- `docker run hello-world`
+
+== Repose Docker Images
+All official Repose Docker images are published on Docker Hub at:
+
+image::http://dockeri.co/image/rackerlabs/repose[Repose Docker,link=https://hub.docker.com/r/rackerlabs/repose/]
+
+== Running a Docker Container
+After you have a running Docker Container Engine, you can run the latest *Repose* release with the following command:
+
+----
+docker run                  \ <1>
+   --detach                 \ <2>
+   --publish 8080:8080      \ <3>
```

Comment:
Using the long-form of options totally makes sense in examples. Not sure why I opted to use the short-form in other places. I like this.

Reply:
I thought they were better for the tutorial too.